src/main/java/com/hillel/evoApp/dao/AdvertisementRepository.java

```diff
@@ -9,6 +9,6 @@
 
 @Repository
 public interface AdvertisementRepository extends JpaRepository<Advertisement, Long> {
-    @Query(value ="SELECT name FROM categories", nativeQuery=true)
+    @Query(value = "SELECT name FROM categories", nativeQuery = true)
```

Comment:
I don't understand why do you get Categories in Advertisement repository? It will be useful to create additional CategoryRepository and get all things there.

Reply:
It was a temporary decision. I will fix it.